apps/docusaurus/docs/guides/randomness.md

```diff
@@ -0,0 +1,314 @@
+---
+title: "Aptos Roll: on-chain public instant randomness API"
+---
```

Comment:
Please update `nextra` as well for randomness.

These are the old docs, which will be replaced.

Reply:
moved.
(or should be copied?)